libs/subliminal_patch/providers/whisperai.py

```diff
@@ -262,6 +282,11 @@ def query(self, language, video):
         else:
             # We must detect the language manually
             detected_lang = self.detect_language(video.original_path)
+            if detected_lang == None:
+                sub.task = "error"
+                # tell the user what is wrong
+                sub.release_info = "bad/missing audio track - cannot transcribe"
```

Comment:
I see you set task to error here but there's no logic in `download_subtitle` that tells it to stop. It will still hit the ASR.

If the user ignores the release_info and clicks the download button, it will cause the provider to error and Bazarr will put it in timeout, right @morpheus65535 ?

Reply:
Actually, it doesn’t hit the ASR because download_subtitle will try to extract the audio stream in the same way as earlier and fail in the same way. It will log an INFO message and return and Bazarr will immediately display an error pop-up without contacting whisper at all.
However, I made a slight improvement to avoid the unnecessary second audio encoding. It will still return early without contacting the ASR.

Reply:
Gotcha. But if it errors, will it prevent whisper provider from working?

Reply:
No. It’s like the whisper provider never even saw the request (because it didn’t)

Reply:
Actually, Bazarr thinks the download succeeded initially. Then an error 500 message appears saying that the downloaded subtitles isn’t valid.
But the key thing is that whisper is not throttled.